file/reader.go

```diff
@@ -47,7 +47,11 @@ func Get(fileContent *Content, opt RenderConfig) (*utils.KongRawState, error) {
 	}
 	builder.defaulter = d
 
-	return builder.build()
+	state, err := builder.build()
+	if err != nil {
+		return nil, errors.Wrap(err, "error building state")
```

Comment:
Can we change the wrap string to `building state`?
As you can see in the output, `error` is printed twice, which is redundant.

Reply:
done